package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kriya",
-  "version": "2.6.1",
+  "version": "2.6.1-rc1",
```

Comment:
You should use `yarn link` ?
You publish every PR right now ?

Reply:
:hankey: :ok_hand: